src/module/data/actor-character.mjs

```diff
@@ -30,10 +28,23 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 			})
 		});
 
-		schema.thorns = new fields.NumberField({
-			integer: true,
+		// should these have some sort of healing pool attached to them?
+		schema.bloodied = new fields.BooleanField();
+		schema.rattled = new fields.BooleanField();
+		schema.conditions = new fields.ArrayField(new fields.StringField());
```

Comment:
Each condition will also need a bit of metadata related to the condition. We would need to be able to note:
- Pools like `4d` for urgent conditions
- Short-term
- Long-term
- Permanent